dotcom-rendering/src/web/components/FilterButton.importable.tsx

```diff
@@ -62,7 +62,7 @@ const valueStyles = css`
 const Label = ({ value, count }: LabelProps) => (
 	<>
 		<span css={valueStyles}>{value}</span>{' '}
-		{count && <span css={countStyles}>({count})</span>}
+		{!!count && <span css={countStyles}>({count})</span>}
```

Comment:
Hey @guardian/journalism-dev I’m assuming here that you don’t want to show labels that have a count of `0`, but if the logic should be different, let me know.